FontraPakMain.py

```diff
@@ -166,6 +166,20 @@ def __init__(self, port):
 
         layout.addWidget(QLabel(f"Fontra version {fontraVersion}"), 4, 0)
 
+        if sys.platform == "darwin":
+            downloadLink = "https://fontra-download.black-foundry.com/FontraPak.dmg"
+        elif sys.platform == "win32":
+            downloadLink = "https://fontra-download.black-foundry.com/FontraPak.zip"
+        else:
+            # We don't have Linux support.
```

Comment:
We totally do. But...

```suggestion
            # We don't provide downloads for other platforms.
```